plugins/oauth/hooks.js

```diff
@@ -201,8 +201,9 @@ module.exports = Object.assign({}, StoreMixin, {
 
   userLoginSuccess() {
     const redirectTo = RouterUtil.getRedirectTo();
+    const isValidRedirect = RouterUtil.isValidRedirect(redirectTo);
```

Comment:
There's a difference between a domain and a url, generally speaking I think that we can't really redirect to a _domain_ only to a URL thus we should change the name

Examples:
dcos.io is a domain name
http://dcos.io is a URL

https://en.wikipedia.org/wiki/URL
https://en.wikipedia.org/wiki/Domain_name